### PR TITLE
Datamodel update

### PR DESCRIPTION
**Pull request by jschummer**
This change is [Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-jore3-importer/90)

### CODE REVIEW

**Review by pkainulainen-vincit — APPROVED** [review]
Looks good to me. However, before you merge this to main, please check if Jarkko found something from this PR because he has some draft comments.

> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-jore3-importer/90#-:-N4kHCFQ53Ch_bkxCm-y:bnptlci)* status: 0 of 61 files reviewed, all discussions resolved (waiting on @pkainulainen-vincit)

**Review by jarkkoka — COMMENTED** [review]
> Reviewed 61 of 61 files at r1, all commit messages.
> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-jore3-importer/90)* status: all files reviewed, 1 unresolved discussion (waiting on @jschummer and @pkainulainen-vincit)

___
*[`src/test/java/fi/hsl/jore/importer/config/jobs/ExportJourneyPatternStopsStepTest.java` line 121 at r1](https://reviewable.io/reviews/hsldevcom/jore4-jore3-importer/90#-N4kN5ZP6smC6ZY_8Kza:-N4kN5ZQ1fxjpncxY8of:b-8519g1) ([raw file](https://github.com/hsldevcom/jore4-jore3-importer/blob/7db08437d0b9c936b0e45789fb82ab7fc9588588/src/test/java/fi/hsl/jore/importer/config/jobs/ExportJourneyPatternStopsStepTest.java#L121)):*
> ```Java
>                 .row(1)
>                 .value(SCHEDULED_STOP_POINT_IN_JOURNEY_PATTERN.SCHEDULED_STOP_POINT_LABEL.getName())
>                 .as("scheduledStopPointLabek")
> ```

Label



_Code quote:_
```Java
Labek
```

**Review by jschummer — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/hsldevcom/jore4-jore3-importer/90)* status: 60 of 61 files reviewed, 1 unresolved discussion (waiting on @jarkkoka and @pkainulainen-vincit)

___
*[`src/test/java/fi/hsl/jore/importer/config/jobs/ExportJourneyPatternStopsStepTest.java` line 121 at r1](https://reviewable.io/reviews/hsldevcom/jore4-jore3-importer/90#-N4kN5ZP6smC6ZY_8Kza:-N4kicldDH_kFSEtJH_K:b-896fix) ([raw file](https://github.com/hsldevcom/jore4-jore3-importer/blob/7db08437d0b9c936b0e45789fb82ab7fc9588588/src/test/java/fi/hsl/jore/importer/config/jobs/ExportJourneyPatternStopsStepTest.java#L121)):*
<details><summary><i>Previously, jarkkoka (Jarkko Kaura) wrote…</i></summary><blockquote>

Label
</blockquote></details>

Done.

**Pull request merged**
